config/settings/rollover.yml

```diff
@@ -0,0 +1,28 @@
+gcp_api_key: please_change_me
```

Comment:
Are we going to inject the values for these via `ENV`?

Reply:
It might be worth checking what's in ENV["SETTINGS__*"] for the other environment so we can cover any missing values

Reply:
Yes, the value is in the app keyvault secret. The only difference between rollover and staging is that staging also has the BigQuery settings. As we aren't going to send data to BQ, that should be ok.